lambda-runtime/src/AWS/Lambda/Client.hs

```diff
@@ -0,0 +1,175 @@
+module AWS.Lambda.Client
+  ( HTTPConfig
+  , InternalLambdaClientError (..)
+  , LambdaClient
+  , LambdaResponse (..)
+  , getHttpConfig
+  , getNextLambdaResponse
+  , sendBootError
+  , sendEvent
+  )
+where
+
+import AWS.Prelude
+import Control.Exception (displayException)
+import Control.Monad.Except (ExceptT(..), liftEither, throwError, withExceptT)
+import System.Environment (lookupEnv)
+
+import qualified Data.Aeson          as JSON
+import qualified Data.ByteString     as BS
+import qualified Network.HTTP.Client as HTTP
+import qualified Network.HTTP.Types  as HTTP
+
+data HTTPConfig = HTTPConfig
+  { request :: HTTP.Request
+  , manager :: HTTP.Manager
+  }
+
+data LambdaResponse = LambdaResponse
```

Comment:
I think this should be named `LambdaEvent`. Yes its a response to the polling API, but its role is the event. We should name this from the perspective of the domain, not the implementation detail its a response to a HTTP request.

Reply:
> I think this should be named `LambdaEvent`. Yes its a response to the polling API, but...

This reasoning makes sense. Will make updates here and elsewhere as pointed out.